splitlock/sl_test.c

```diff
@@ -0,0 +1,131 @@
+// SPDX-License-Identifier: GPL-2.0-only
+// Copyright (c) 2022 Intel Corporation.
+
+/*
+ * Split lock test app to check if the locked instruction will trigger #AC
+ *
+ */
+
+#define _GNU_SOURCE
+#include <sys/types.h>
+#include <sys/stat.h>
+#include <sys/wait.h>
+#include <fcntl.h>
+#include <stdlib.h>
+#include <signal.h>
+#include <stdio.h>
+#include <unistd.h>
+#include <string.h>
+#include <time.h>
+#include <sys/time.h>
+//#include <stdbool.h>
+#include <stdio.h>
+
+void catch_sigbus(int sig)
+{
+	printf("Caught SIGBUS/#AC due to split locked access\n");
+
+	exit(-1);
```

Comment:
confirm the exit code -1 or  0 ?

Reply:
catch_sigbus is a void function, it is to handle user space SIGBUS which is sent by kernel. For test purpose, print out a message is enough. To mark it is abnormal, use non zero is OK, whatever it is 1 or -1.